hlsbounds:
    lowerbound:
        h: 25.0
        l: 0.00
        s: 0.30
    upperbound:
        h: 67.0
        l: 0.90
        s: 1.00
lineparams:
    minLineLength: 60
    maxLineGap: 40
